gsmphys/sfc_ocean.f

```diff
@@ -138,6 +139,11 @@ subroutine sfc_ocean                                              &
           tem      = 1.0 / rho
           hflx(i)  = hflx(i) * tem * cpinv
           evap(i)  = evap(i) * tem * hvapi
+          if (evap(i) .lt. -maxevap(i)) then
+             chh(i) = -maxevap(i)/evap(i)*chh(i)
+             hflx(i) = -maxevap(i)/evap(i)*hflx(i)
```

Comment:
Checking order of operations is supposed to be `(maxevap/evap) * _var_` and you want to negate it